dashboard/app/models/authentication_option.rb

```diff
@@ -123,4 +123,16 @@ def summarize
       errors.add :email, I18n.t('errors.messages.taken')
     end
   end
+
+  private def cred_type_and_auth_id_must_be_unique
+    # skip the db lookup if possible
+    return unless authentication_id.present? &&
```

Comment:
Is there a credential type for which we expect a blank `authentication_id`?

Reply:
It should always be present, although because the database column doesn't specify `not null`, I added this for safety + the possibility that future `AuthenticationOption` types might not have authentication ids